python/hail/methods/tests.py

```diff
@@ -82,6 +82,47 @@ def compare(ds, min=None, max=None):
         methods.ibd(dataset, dataset['dummy_maf'], min=0.0, max=1.0)
         methods.ibd(dataset, dataset['dummy_maf'].to_float32(), min=0.0, max=1.0)
 
+    def test_impute_sex_same_as_plink(self):
+        import subprocess as sp
+
+        ds = methods.import_vcf(test_file('x-chromosome.vcf'))
+
+        sex = methods.impute_sex(ds.GT, include_par = True)
+
+        vcf_file = utils.get_URI(utils.new_temp_file(prefix="plink", suffix="vcf"))
+        out_file = utils.get_URI(utils.new_temp_file(prefix="plink"))
+
+        methods.export_vcf(ds, vcf_file)
+
+        try:
+            out = sp.check_output(
+                ["plink", "--vcf", vcf_file, "--const-fid", "--check-sex",
+                 "--silent", "--out", out_file],
+                stderr=sp.STDOUT)
```

Comment:
Can you add a test using the alternate allele frequency expression computed from call_stats.AF and make sure your results agree with your formula with aaf=None?